internal/aws/k8s/k8sclient/node_info.go

```diff
@@ -5,15 +5,24 @@ package k8sclient // import "github.com/open-telemetry/opentelemetry-collector-c
 
 import (
 	v1 "k8s.io/api/core/v1"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/k8s/k8sutil"
+)
+
+type Label string
+
+const (
+	SageMakerNodeHealthStatus Label = "sagemaker.amazonaws.com/node-health-status"
+	SageMakerNodeHealthStatusSC Label = "n"
```

Comment:
Lets add a comment saying this is a single char to reduce memory consumption? This name doesnt matter, it could be a single byte as well.